sympy/physics/vector/point.py

```diff
@@ -471,6 +472,36 @@ def v2pt_theory(self, otherpoint, outframe, fixedframe):
         self.set_vel(outframe, v + (omega ^ dist))
         return self.vel(outframe)
 
+    def _calc_vel(self, frame, pos_vect):
+        if len(self._pos_dict) == 1:
+            return False
+        self._vel_dict_pos = self._vel_dict.copy()
+        for p, p_pos in self._pos_dict.items():
+            try:
+                p_vel = p._vel_dict[frame]
+                p_pos.dt(frame)
```

Comment:
try/except clauses should ideally handle only the errors from one statement. So:

```
try:
    p._vel_dict[frame]
except KeyError:
    ...
else:
    p_pos.dt(frame)
```

Reply:
Both statements would give different errors that's why it has 2 except clauses

Reply:
You should catch the errors from different statements in different try clauses then.